generated_golden.sh

```diff
@@ -54,6 +57,11 @@ scaffold_test_project() {
 		$kb alpha webhook --group core --version v1 --kind Namespace --type=mutating --operations=update --make=false
 		$kb create api --group policy --version v1beta1 --kind HealthCheckPolicy --example=false --controller=true --resource=true --namespaced=false --make=false
 	elif [ $version == "2" ]; then
+        export GO111MODULE=on
+        go get sigs.k8s.io/controller-tools/cmd/controller-gen@crdgeneratortmp
```

Comment:
we can use the make target to install controller-gen ? (or may be we should invoke that as part of project-init now ? )